src/Uno.UI/Microsoft/UI/Xaml/Controls/ColorPicker/ColorPickerSlider.cs

```diff
@@ -276,8 +275,7 @@ private string GetToolTipString()
 							throw new InvalidOperationException("Invalid ColorPickerHsvChannel."); // Uno Doc: 'throw winrt::hresult_error(E_FAIL);'
 					}
 
-					return string.Format(
-						CultureInfo.CurrentUICulture,
+					return StringUtil.FormatString(
 						localizedString,
```

Comment:
Let me know if I should introduce a new overload of `StringUtil.FormatString` that takes `IFormatProvider`

Reply:
Yes, I think this is a good idea. Actually, I think StringUtil.FormatString() needs to always require an IFormatProvider so I would modify the existing method. string.Fromat (which is used internally) always requires a format provider by standard convention anyway (most analyzers will complain if you forget to specify one)

`ColorPicker` is using integers which aren't usually localized; however, numbers should be using a culture info format provider just in this case. Additionally, this should NOT be using the current thread culture and should be using the UI culture as default since it's a localized resource for display only.